modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -1269,6 +1311,31 @@ void cv::gimpl::GFluidExecutable::run(std::vector<InObj>  &&input_objs,
     }
 }
 
+cv::gimpl::GParallelFluidExecutable::GParallelFluidExecutable(const ade::Graph                      &g,
+                                                              const FluidGraphInputData             &graph_data,
+                                                              const std::vector<GFluidOutputRois>   &parallelOutputRois)
+{
+    for (auto&& rois : parallelOutputRois){
+        tiles.emplace_back(g, graph_data, rois.rois);
+    }
+}
+
+
+void cv::gimpl::GParallelFluidExecutable::reshape(ade::Graph&, const GCompileArgs& )
+{
+    //FIXME: do we need it ?
```

Comment:
Yes, we need it.

In fact, _all_ our current plugins are can support reshaping, and it will become especially important when inference & streaming come to the picture.

Reply:
changed `TODO`